ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -707,7 +707,11 @@ final class Metrics(val registry: MetricRegistry) {
       private val Prefix: MetricName = daml.Prefix :+ "http_json_api"
 
       val httpRequest: Timer = registry.timer(Prefix :+ "http_request")
+      val httpRequestRunning: Meter = registry.meter(Prefix :+ "http_request_running")
       val websocketRequest: Counter = registry.counter(Prefix :+ "websocket_request")
+      val commandSubmission: Meter = registry.meter(Prefix :+ "command_submission")
+      val packageAllocation: Meter = registry.meter(Prefix :+ "package_allocation")
+      val partyAllocation: Meter = registry.meter(Prefix :+ "party_allocation")
```

Comment:
this seems somewhat inconsistent. For requests you suffix the concurrent counter with `_running`. For everything else you don’t. For websockets, `_request` is a counter for http it’s a timer.
I think what would help is some consistency and maybe some comments here on what those metrics represent.

Reply:
you're right, I oversaw that one. I'll change that.

Reply:
addressed with ce07e09

Reply:
I could also rename the timers with a suffix `_timing`.

Reply:
I think I'm doing that as I prefer better names right now.

Reply:
ok done with 0a40a8b